doc/classes/XRPose.xml

```diff
@@ -14,7 +14,7 @@
 		<method name="get_adjusted_transform" qualifiers="const">
 			<return type="Transform3D" />
 			<description>
-				Returns the [member transform] with world scale and our reference frame applied. This is the transform used to position [XRNode3D] objects.
+				Returns the [member transform] with world scale and our reference frame applied. This is the transform used to position [XRNode3D] objects. Only the origin is adjusted, not the basis.
```

Comment:
This doesn't seem to be true from the code though:
https://github.com/godotengine/godot/blob/5034478611697358f4e135c69f364ef6d8cd54dc/servers/xr/xr_pose.cpp#L99

It transforms the result by the reference frame

Reply:
If OP is confident on this there may be a bug either in Godot itself or their hardware, I wonder.

Reply:
Might be that their testing conditions has a specific reference frame case that doesn't showcase all outcomes, but this code clearly modifies the full transform